MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -7888,7 +7884,7 @@ private String doMaintenanceOnUnitPart(Unit u, Part p, Map<Part, Integer> partsT
         partReport += " rolled a " + roll + ", margin of " + margin;
 
         switch (p.getQuality()) {
-            case Part.QUALITY_A: {
+            case A: {
```

Comment:
I’m a little concerned that just having ‘A’, etc, instead of ‘QUALITY_A’ will lead to confusion down the line.